user-guide/Basic_Functionality/Protocols_and_templates/Alarm_templates/Configuring_alarm_templates/Configuring_anomaly_detection_alarms.md

```diff
@@ -70,19 +70,27 @@ To configure anomaly detection alarms for specific parameters<!--RN 37171 + 3714
         > [!NOTE]
         > Any customized behavioral anomaly monitoring setup containing relative or absolute thresholds will be lost if you downgrade to DataMiner version 10.3.11 or older. These versions do not support this extended anomaly configuration, and the thresholds will be automatically determined by DataMiner Analytics, similar to the default *Smart* option<!--RN 37434-->.
 
-1. Click *Close* in the lower right corner to exit the *Anomaly Alarm Settings* pop-up window.
+1. From DataMiner 10.4.12/10.5.0 onwards<!-- RN 40837+41017 -->, in the *Proactive alarm setting* section, optionally configure the alarms that should be triggered based on [proactive cap detection](xref:Proactive_cap_detection):
 
-   In the alarm template editor, depending on your changes, the button in the *Anomalies* column will now indicate the level of anomaly monitoring that has been configured:
+   1. Select the checkbox.
 
-   - **Disabled**: No anomaly alarms are generated for this parameter.
+   1. Select the severity of the generated alarms: *Warning*, *Minor*, *Major*, or *Critical*.
 
-   - **Customized**: You have customized the anomaly alarm settings for this parameter.
+   1. Select the trend prediction range that will trigger the alarms: *Upper range (100.0 %)*, *Critical high (90.0 %)*, *Major High (80.0 %)*, or *Lower range (0.0 %)*.
```

Comment:
I think this will have to be explained a bit more: the value between brackets is not always 100% or 90%, nor are these the only possible options. 

- Upper and lower range refers to the range values specified in the protocol (or in case of percentages and no range value specified, we assume 100% is the upper range and 0% the lower range). They are only shown when these ranges are specified, and in that case the value between brackets corresponds to these range values. 
- Critical / Major / Minor / Warning, each available in High and Low variant, refer to the alarm thresholds set in the main template. They will only be shown if there is actually a threshold configured in the main template, and again the value between brackets is the configured threshold. For instance, if  a critical high alarm threshold of 100 Kbps is configured in the alarm template for an Audio Bit Rate parameter, then in this pop-up for that parameter, there will be an option 'Critical high (100 Kbps)'